src/validation.cpp

```diff
@@ -2145,6 +2145,7 @@ bool CChainState::FlushStateToDisk(
             if (!CoinsTip().Flush())
                 return AbortNode(state, "Failed to write to coin database");
             nLastFlush = nNow;
+            TRACE4(flush, coins_flushed, nNow.count(), mode, coins_count, coins_mem_usage);
```

Comment:
Could be multi-line, similar to the other tracepoints, too. And you likely want to cast here too.

Reply:
Context shouldn't be `flush`. I think `utxocache` is clearer.

Reply:
Made the refactoring, also added a new parameter of `outpoint.hash`